arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -1791,7 +1855,32 @@
         }
       },
       "x-ms-azure-resource": true
-    }
+    },
+	"Sku": {
+	 "properties": {
```

Comment:
What are the required properties of "Sku"?

Can name, tier, family, capacity, size be enums? Do they have a finite set of values? How will the user know what values to provide?